### PR TITLE
Add tf/ERWR

### PR DESCRIPTION
**Pull request by CatherineSue**
1. Add tf/ERWR.
2. Fix warning in garage.baselines.LinearFeatureBaseline.
3. Set seed in TfGraphTestCase to avoid large flakiness.

### CODE REVIEW

File: garage/tf/algos/erwr.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+                 positive_adv=None,
+                 **kwargs):
+        if optimizer is None:
+            optimizer = LbfgsOptimizer
+            if optimizer_args is None:
+                optimizer_args = dict()
+        super(ERWR, self).__init__(
+            optimizer=optimizer,
+            optimizer_args=optimizer_args,
+            positive_adv=True if positive_adv is None else positive_adv,
```
Comment (ryanjulian) [inline garage/tf/algos/erwr.py]: Isn't this equivalent to `positive_adv=True` above?

Comment (CatherineSue) [reply garage/tf/algos/erwr.py]: Yes. Addressed.

File: garage/tf/algos/erwr.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    def __init__(self,
+                 optimizer=None,
+                 optimizer_args=None,
+                 positive_adv=True,
+                 **kwargs):
+        if optimizer is None:
+            optimizer = LbfgsOptimizer
+            if optimizer_args is None:
+                optimizer_args = dict()
+        super(ERWR, self).__init__(
```
Comment (ahtsan) [inline garage/tf/algos/erwr.py]: just a comment: super().__init__ also works

Comment (ryanjulian) [reply garage/tf/algos/erwr.py]: super().__init__() is the preferred style in Python 3

**Pull request merged**